code/datums/martial.dm

```diff
@@ -451,24 +454,24 @@
 		D.adjustBrainLoss(25)
 	return 1
 
-/datum/martial_art/CQC/proc/Pressure(mob/living/carbon/human/A, mob/living/carbon/human/D)
+/datum/martial_art/cqc/proc/Pressure(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	D.visible_message("<span class='warning'>[A] forces their arm on [D]'s neck!</span>")
 	D.adjustStaminaLoss(60)
 	playsound(get_turf(A), 'sound/weapons/cqchit1.ogg', 50, 1, -1)
 	return 1
 
-/datum/martial_art/CQC/proc/Restrain(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	if(restrained)
+/datum/martial_art/cqc/proc/Restrain(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(restraining)
 		return
 	if(!D.stat)
 		D.visible_message("<span class='warning'>[A] locks [D] into a restraining position!</span>", \
 							"<span class='userdanger'>[A] locks you into a restraining position!</span>")
 		D.adjustStaminaLoss(20)
 		D.Stun(5)
```

Comment:
And while we're here, maybe we can make this automatically upgrade your grab to a chokehold.
```DM
if(A.grab_state < GRAB_NECK)
	A.grab_state = GRAB_NECK
```

Make sure to change [this ```if(restraining)```](https://github.com/tgstation/tgstation/pull/21201/files#diff-76481ba91c3b460c7445e4d8cf8859e4R464) to ```if(restraining && A.pulling = D)``` though.